File: app/src/main/java/com/talktiva/pilot/model/Slider.java
package com.talktiva.pilot.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Slider implements Serializable {

    @SerializedName("text")
    private Integer text;

    @SerializedName("image_url")
    private String imageUrl;

    public Slider(Integer text, String imageUrl) {
        this.text = text;
        this.imageUrl = imageUrl;
    }

    public Integer getText() {
        return text;
    }

    public String getImageUrl() {
        return imageUrl;
    }
}

File: app/src/main/java/com/talktiva/pilot/model/Slider.kt
package com.talktiva.pilot.model

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class Slider(@field:SerializedName("text")
             var text: Int?, @field:SerializedName("image_url")
             var imageUrl: String) : Serializable
